Jenkinsfile

```diff
@@ -11,7 +10,7 @@ pipeline {
                 axes {
                     axis {
                         name 'BUILD'
-                        values 'php56|php70', 'php71|php72', 'php73|php74', 'php80'
+                        values 'php56', 'php70', 'php71', 'php72', 'php73', 'php74', 'php80'
```

Comment:
was this in just for debugging as I saw before or here to stay?

Reply:
I had better remove it

Reply:
I'll keep the removal of parallelsAlwaysFailFast() though, as it's hard to find out why something failed with that on.